catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -109,6 +111,7 @@ public RegistryStoreImpl(EncryptionService encryptionService) {
     @Override
     protected Map<String, Consumer<Object>> getAdditionalConsumers() {
         Map<String, Consumer<Object>> map = new HashMap<>();
+        map.put(REGISTRY_URL, value -> setRegistryUrl((String) value));
         map.put(AUTO_PUSH, value -> setAutoPush((Boolean) value));
         map.put(REMOTE_NAME, value -> setRemoteName((String) value));
```

Comment:
Shouldn't need the REMOTE_NAME anymore since we should be setting the ID to remote registries name. This is the class that should have the duplication logic in it.